get.sh

```diff
@@ -101,6 +101,19 @@ getPackage() {
             echo
             echo "Running with sufficient permissions to attempt to move $REPO to $BINLOCATION"
 
+            if [ ! -w "$BINLOCATION/$REPO" ] && [ -f "$BINLOCATION/$REPO" ]; then
+
+            echo
+            echo "================================================================"
+            echo "==  $BINLOCATION/$REPO already exists and is not writeable  =="
+            echo "==  by the current user.  Please adjust the binary ownership  =="
+            echo "==  or run with sudo:  curl -SLs get.k3sup.dev | sudo sh      ==" 
```

Comment:
we need to give `https://` in the URL to force TLS

Reply:
`or run with sudo` is probably sufficient though without the URL, that the user already has